system/core-workflows/how-do-they-work.rst

```diff
@@ -189,15 +190,20 @@ unset readonly |ui|
 Stop after match
 ----------------
 
-Stop evaluation of other, following workflows that would match otherwise.
+Here you can decide if other workflows are executed after the current one.
+
+If set to ``no`` (default), no further worflows will be executed after the
+current one.
 
-Default: ``no``
+If set to ``yes``, further workflows will be executed if they match the
+condition. In this case, it is possible that your actions from the current
+workflow can be overwritten by another workflow.
```

Comment:
Something sounds strange here, it may be the opposite meaning. `Stop after match: no` should allow further workflows to be executed, since there will be no stopping after the match. `Stop after match: yes` should skip further workflows.

Reply:
Right. Changed it already in https://github.com/zammad/zammad-admin-documentation/pull/370 as well.